server/container/guice/cassandra-guice/src/test/java/org/apache/james/JamesServerWithRetryConnectionTest.java

```diff
@@ -34,11 +34,13 @@
 import java.nio.charset.Charset;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class JamesServerWithRetryConnectionTest {
     private static final int IMAP_PORT = 1143;
+    private static final long WAITING_TIME = TimeUnit.SECONDS.convert(30, TimeUnit.MILLISECONDS);
```

Comment:
Do you want to transforme 30 to 30000 (converting from second to ms) or 30 to 3e-2 (converting from ms to second) ?